api/src/main/java/io/minio/messages/Item.java

```diff
@@ -86,7 +87,9 @@ public String objectName() {
 
   /** Returns last modified time of the object. */
   public ZonedDateTime lastModified() {
-    return lastModified.zonedDateTime();
+    return Optional.ofNullable(lastModified)
```

Comment:
@hwaite I suspect a server bug. Please share the reproducer along with your object storage server information.

Reply:
See #1606.